packages/flutter_tools/test/web.shard/test_data/hot_restart_web_test_common.dart

```diff
@@ -127,7 +115,5 @@ Future<void> _testProject(
         await subscription.cancel();
       }
     },
-    // Skipped for https://github.com/flutter/flutter/issues/110879.
```

Comment:
Is this test not running canvaskit anymore?

Reply:
The opposite :)

1. The ` - canvaskit` suffix in test name:

We used to have a test for HTML and a test for CanvasKit:
https://github.com/flutter/flutter/blob/8e0993eda8b0bbf0e11162cf18df1effd134db09/packages/flutter_tools/test/web.shard/hot_reload_web_test.dart#L56-L94

The HTML test was removed, so it doesn't make sense to have the ` - canvaskit` suffix anymore.

2. The `skip: true`:

This was skipped a [long time ago](https://github.com/flutter/flutter/pull/86885), I thought it may have been fixed already. I ran the test locally multiple times and passed consistently.